ios/RCTOneSignal/RCTOneSignalEventEmitter.m

```diff
@@ -383,13 +396,14 @@ + (void)sendEventWithName:(NSString *)name withBody:(NSDictionary *)body {
     [OneSignal removeTriggerForKey:key];
 }
 
+// to do, check this still works
```

Comment:
Looks like we still need to test this.

Reply:
don't recall why I commented this

Reply:
reason was I switched the `RCT_REMAP_METHOD` to `RCT_EXPORT_METHOD` but it has to stay as is

Reply:
I don't see a change here, looks like it has been and still is `RCT_REMAP_METHOD`. 
Since nothing changed with this it seems like we can just remove this comment then.